.github/workflows/build_test_notebooks.yml

```diff
@@ -7,40 +8,45 @@ on:
     branches:
       - tox_update
   workflow_dispatch:
+
 jobs:
   build-notebooks:
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout stingray repo
         uses: actions/checkout@v4
         with:
           repository: StingraySoftware/stingray
           path: ./stingray
+
       - name: Checkout notebooks repo
```

Comment:
Is this step needed?

Reply:
No, I fixed the branch in the latest commit so this is not required here, thank you!